src/components/App.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import './styles/App.scss';
+
+function App() {
```

Comment:
I personally like functions better because I think [React Hooks](https://reactjs.org/docs/hooks-intro.html) are more flexible and clearer (for more complex functionality at least) than having to rely on stuff like `componentDidMount` and `componentWillUpdate` and yadda yadda. [This doc page](https://reactjs.org/docs/hooks-effect.html) was what convinced me to make the switch lol. I think I'll keep it with functions for now, but if the majority of the team prefers classes I'll change it